src/app/custom-input-error.component.ts

```diff
@@ -0,0 +1,23 @@
+import {Component} from "@angular/core";
+import {JsonPipe, NgFor, NgIf} from "@angular/common";
+import {injectErrorField} from "@signal-form/signal-input-error.directive";
+
+@Component({
+  selector: 'custom-signal-input-error',
+  standalone: true,
+  imports: [
+    NgIf,
+    NgFor,
+    JsonPipe
+  ],
+  template: `
+    <div *ngIf="touchedState() === 'TOUCHED'">
+      <p *ngFor="let error of errors()">{{error | json}}</p>
+    </div>
+  `
+})
+export class CustomErrorComponent {
+  private _formField = injectErrorField();
+  public touchedState = this._formField.touchedState;
+  public errors = this._formField.errorsArray;
```

Comment:
It would be better to use `errors` here.
The idea behind `errorsArray` is that is also includes (grand-)child errors, the `path` defines which component has errors.
While `errors` are the errors of the form field.


```suggestion
  public errors = this._formField.errors;
```

Reply:
That makes sense! I had incorporated the message as part of the errorsArray so each error in the array looks like this:
```json
  {
    "path": "passwords.password",
    "key": "minLength",
    "details": {
      "currentLength": 0,
      "minLength": 5
    },
    "message": "Password must be at least 5 characters long. Add at least 5 characters"
  },
```
I think the errors signal mimics what we get from Angular right now without any of the "metadata".
Would you be open to changing it to something like:
```json
{
 "required": {
    "details": true,
    "message": "Password is required"
  },
  "minLength": {
    "details": {
      "currentLength": 0,
      "minLength": 5
    },
    "message": "Password must be at least 5 characters long. Add at least 5 characters"
  },
}
```
That way we expose the message directly in the errors subject. Let me know what you think 👍